tpot/tpot.py

```diff
@@ -214,10 +214,10 @@ def fit(self, features, classes):
             np.random.shuffle(data_columns)
             training_testing_data = training_testing_data[data_columns]
 
-            training_indices, testing_indices = next(iter(StratifiedShuffleSplit(training_testing_data['class'].values,
-                                                                                 n_iter=1,
+            training_indices, testing_indices = train_test_split(training_testing_data.index,
```

Comment:
Yes, I have tested it on IRIS and MNIST dataset. Works the same. We can also do it the way you have pointed out, but using `training_testing_data.index` to get `training_indices, testing_indices` is in line with rest of our code format.